master/internal/rm/kubernetesrm/pods.go

```diff
@@ -133,6 +134,7 @@ type reattachPodResponse struct {
 
 // Initialize creates a new global pods actor.
 func Initialize(
+	ctx *actor.Context,
```

Comment:
Is there a reason this wasn't included in the first place? And if so, should a different context be used to send the warning if a permission error occurs during `p.startNodeInformer`?